source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,132 @@
+
+By default the Terminal & SSH addon provides a web terminal which can be added as an item to your side-panel. 
+
+---
+
+### Installing and using the Visual Studio Code (VSC) add-on
+
+The Visual Studio Code add-on provides access through a feature packed web-based version of the VSC editor and currently only supports AMD64 and aarch64/ARM64 machines. The add-on also provides access to the Home Assistant Command Line Interface (CLI) using VSC's built in terminal, which allows for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring snapshots, and more. (See [Home Assistant via Command Line](https://www.home-assistant.io/hassio/commandline/) for further info).
+
+There is no configuration required for editing files within your /config directory. In order to enable access to further directories, editing the add-on configuration is required. See the add-on documentation for details. 
+
+---
+
+### Installing and using the File Editor add-on
+
+A more basic and light weight alternative to Visual Studio Code, the File Editor add-on provides access through Hass-Configurator, which is a web-based filesystem-browser and text-editor. YAML files are automatically checked for syntax errors while editing.
+
+There is no configuration required for editing files within your /config directory. In order to enable access to further directories, editing the add-on configuration is required. See the add-on documentation for details. 
+
+---
+
+## Home Assistant OS Snapshots
+
+Snapshots are a compressed archive file (.tar) of your Home Assistant configuration, in addition to any add-on data and configuration. These are easily made in Home Assistant OS from the Supervisor Snapshot panel and are also available as services which can be used in automations. They can be created from the UI or the CLI (see below) and are stored in the /backup directory.
+
+A full snapshot includes the following directories:
+
+* config
+* share
+* add-ons (these are manually installed or created add-ons, and not those installed from the store)
+* ssl
+* media
+
+A partial snapshot consists of any number of the above default directories and installed add-ons.
+
+### Making a Snapshot from the UI
+
+1. Go to Supervisor > Snapshots in the UI
+2. Provide a name for the snapshot.
+3. Choose full or partial.
+4. Choose to password protect or not. Password protected snapshots cannot easily be browsed outside of Home Assistant OS
+5. Click "Create" to begin the snapshot.
+
+### Restoring a Snapshot on a new install
+
+Restoring a snapshot on a fresh install of Home Assistant OS requires that you first upload the snapshot to the backup directory on the new instance. From the Supervisor Snapshot panel, these can be uploaded using the following steps:
```

Comment:
in 2 days when 0.116 drops this entire section should be rewritten since you now can do that from the onboarding.